libslide/src/scanner.rs

```diff
@@ -1,23 +1,23 @@
 mod types;
-use types::TokenType;
 use types::Token;
+use types::TokenType;
 
 pub struct Scanner {
-    input: Vec<char>, 
-    pub output: Vec<Token>
+    input: Vec<char>,
+    pub output: Vec<Token>,
 }
 
```

Comment:
I don't know. This is Rust style convention; presumably, it makes adding a new item to the end of the list easier because you don't have to add a comma to the one before